layer/USAGE_android.md

```diff
@@ -122,31 +126,30 @@ instances consecutively, it will be necessary to enable capture file timestamps
 to prevent each new instance from overwriting the file created by the previous
 instance.
 
+If the layer fails to open the capture file, it will make the call to
+`vkCreateInstance` fail, returning `VK_ERROR_INITIALIZATION_FAILED`.
+
 ### Specifying Capture File Location
 The capture file's save location can be specified by setting the
 `debug.gfxrecon.capture_file` system property, described above in
-the [GFXRecon Debug Options](#gfxrecon-debug-options) section.
-
-The property may be set through ADB with the following command:
-```
-adb shell "setprop debug.gfxrecon.capture_file '<path_to_file>'"
-```
+the [Layer Options](#layer-options) section.
 
 ### Timestamps
-When capture file timestamps are enabled, a timestamp with an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) based
+When capture file timestamps are enabled, a timestamp with an
+[ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) based
 format will be added to the name of every file created by the layer. The
 timestamp is generated when the capture file is created by the layer's
 `vkCreateInstance` function and is added to the base filename specified
-through the `debug.gfxrecon.capture_file` system property. Timestamped
-filenames have the form:
+through the `debug.gfxrecon.capture_file` system property. Timestamps have
+the form:
  ```
-gfxrecon_capture_yyyymmddThhmmss.gfxr
+yyyymmddThhmmss
 ```
-
 where the lower-case letters stand for: Year, Month, Day, Hour, Minute, Second.
 The `T` is a designator that separates the date and time components. Time is
 reported for the local timezone and is specified with the 24-hour format.
 
-The following example shows a timestamp for a file that was created at 2:35 PM
+The following example shows a timestamp that was added to a file that was
+originally named `gfxrecon_capture.gfxr` and was created at 2:35 PM
```

Comment:
Just making sure... I interpret this to mean that, if timestamps are on, and my filename is `bob` (with no suffix), then the timestamped file is `bob_yyyymmddThhmmss` (still with no suffix); and if my filename is `special.bob` (i.e. with a `.bob` suffix), then the timestamped file is `special_yyyymmddThhmmss.bob` (still with a `.bob` suffix), yes?

Reply:
Yes, that is correct.